### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by Moussiaa**

### CODE REVIEW

**Comment by andrewoid** [comment]
Screenshot needs to be in the description of the PR.

File: src/main/java/adry/projectile/Main.java [hunk 1]
```diff
@@ -6,3 +6,5 @@ public static void main (String args[]) {
         frame.setVisible(true);
     }
 }
+
+//
```
Comment (andrewoid) [inline src/main/java/adry/projectile/Main.java]: empty comment

File: src/main/java/adry/projectile/ProjectileFrame.java [hunk 2]
```diff
@@ -110,34 +124,39 @@ public void actionPerformed(ActionEvent e) {
                         projectile.setSeconds(
                                 Double.parseDouble(secondsField.getText())
                         );
-                        interceptXlabel.setText(Double.toString(projectile.getX()));
-                        peakYlabel.setText(Double.toString(projectile.getY()));
+                        emptyX.setText(Double.toString(projectile.getX()));
+                        emptyY.setText(Double.toString(projectile.getY()));
                         fieldY.setText(Double.toString(projectile.getPeakY()));
                         fieldX.setText(Double.toString(projectile.getInterceptX()));
```
Comment (andrewoid) [inline src/main/java/adry/projectile/ProjectileFrame.java]: You should find better names for your variables so they describe what they are.

File: src/main/java/adry/projectile/ProjectileFrame.java [hunk 3]
```diff
@@ -110,34 +124,39 @@ public void actionPerformed(ActionEvent e) {
...
+                        calculate();
+                        //getContentPane().add(new ProjectileGraph(projectile.getPeakY(), projectile.getInterceptX());
```
Comment (andrewoid) [inline src/main/java/adry/projectile/ProjectileFrame.java]: remove commented out code.

File: src/main/java/adry/projectile/ProjectileFrame.java [hunk 4]
```diff
@@ -110,34 +124,39 @@ public void actionPerformed(ActionEvent e) {
                         projectile.setSeconds(
```
Comment (andrewoid) [inline src/main/java/adry/projectile/ProjectileFrame.java]: The only thing you should be doing in the actionPerformed method is calling `calculate()` like you do in the other Listeners.

**Comment by Moussiaa** [comment]
projectile graph

**Comment by Moussiaa** [comment]
projectile graph

File: src/main/java/adry/projectile/ProjectileFrame.java [hunk 5]
```diff
@@ -103,41 +117,33 @@ public void update(DocumentEvent e) {
...
-                Double.parseDouble(secondsField.getText())
-        );
-        interceptXlabel.setText(Double.toString(projectile.getX()));
-        peakYlabel.setText(Double.toString(projectile.getY()));
-        fieldY.setText(Double.toString(projectile.getPeakY()));
-        fieldX.setText(Double.toString(projectile.getInterceptX()));
+        try {
+            Projectile projectile = new Projectile(
+                    Double.parseDouble(velocityField.getText()),
+                    angleAdjuster.getValue()
```
Comment (andrewoid) [inline src/main/java/adry/projectile/ProjectileFrame.java]: It should be angle then velocity.

**Review by andrewoid — APPROVED** [review]

**Pull request merged**